src/PageGenerator/project.json

```diff
@@ -5,18 +5,11 @@
     "url": "git://github.com/aspnet/diagnostics"
   },
   "dependencies": {
-    "Microsoft.AspNet.Diagnostics.Elm": "1.0.0-*",
-    "Microsoft.AspNet.Diagnostics.Entity": "7.0.0-*",
     "Microsoft.AspNet.Razor": "4.0.0-*"
   },
 
   "frameworks": {
-    "dnx451": { },
-    "dnxcore50": {
```

Comment:
It seems odd to me that we can't write an app that uses Razor that runs on both CLRs...